Cargo.toml

```diff
@@ -63,6 +63,7 @@ bee-ternary = { version = "0.5.2", default-features = false }
 
 [dev-dependencies]
 dotenv = { version = "0.15.0", default-features = false }
+env_logger = { version = "0.9.0", default-features = false }
```

Comment:
We already have `fern-logger` as dependency, you should be able to do the same things with it